http-server/src/test/java/org/triplea/server/access/BannedPlayerFilterTest.java

```diff
@@ -104,8 +104,9 @@ private void verifyForbiddenRequest() {
 
       final Response response = responseCaptor.getValue();
       assertThat(response.getStatus(), is(Status.UNAUTHORIZED.getStatusCode()));
-      assertThat((String) response.getEntity(), StringContains.containsString("5 minutes"));
-      assertThat((String) response.getEntity(), StringContains.containsString(BAN_ID));
+      assertThat(
+          response.getStatusInfo().getReasonPhrase(), StringContains.containsString("5 minutes"));
+      assertThat(response.getStatusInfo().getReasonPhrase(), StringContains.containsString(BAN_ID));
```

Comment:
Usually we statically import those methods, don't we?

Reply:
Updated: 34d959a6a